mmdeploy/backend/rknn/__init__.py

```diff
@@ -13,15 +13,26 @@ def is_available():
     return importlib.util.find_spec('rknn') is not None
 
 
+def package_info():
+    import pkg_resources
+    for p in pkg_resources.working_set:
+        if p.project_name.startswith('rknn-toolkit'):
+            return dict(name=p.project_name, version=p.version)
+    return dict(name=None, version=None)
+
+
 def device_available():
     """Check whether device available.
 
     Returns:
         bool: True if the device is available.
     """
-    ret = subprocess.check_output('adb devices', shell=True)
-    match = re.search(r'\\n\w+\\tdevice', str(ret))
-    return match is not None
+    try:
+        ret = subprocess.check_output('adb devices', shell=True)
```

Comment:
what if the connected device is not a rockchip device?

Reply:
It's a behavior controlled by user. Any idea to improve the usage?

Reply:
I think device check is not necessary.

Reply:
Okay, will delete the log info of adb device, which is a bonus actually. The function was designed to skip UT in CI since CI can install rknn-toolkit but could not connect a device.

Reply:
I don't think we should keep this function. As @lvhan028 mentioned, the device might still be available even when `device_available()==False`.